modules/gridmap/editor/grid_map_editor_plugin.h

```diff
@@ -92,23 +92,30 @@ class GridMapEditor : public VBoxContainer {
 	List<SetItem> set_items;
 
 	GridMap *node = nullptr;
+	// caching the node global transform to detect when the node has been
+	// moved/scaled/rotated.
```

Comment:
Unsure if this comment is necessary but:
```suggestion
	// Cache the node's global transform to detect when the node has been moved/scaled/rotated.
```